### DISCUSSION TITLE
How to connect CKAN Database (PostgreSQL) to Navicat (or any DBMS)

### DISCUSSION THREAD

**Discussion by plenthoz** [ckan/ckan#7315] [Q&A]
Hy everyone, i tried to connect CKAN database with Navicat but no table show up

how can i open CKAN database via Database Management System?

**Comment by pdelboca** [comment]
Hello @plenthoz !

CKAN has a normal Postgresql database so there is nothing impeding to connect a DBMS to it.

If no tables are shown, are you sure that the Database has been properly initialized? https://docs.ckan.org/en/2.9/maintaining/installing/install-from-source.html?highlight=db%20init#create-database-tables
